paddle/gserver/layers/MaxLayer.h

```diff
@@ -26,6 +26,11 @@ namespace paddle {
  * If SequenceLevel = kNonSeq:
  *    Output: output size is the number of input sequences (NOT input instances)
  *    output[i] = max_{for each instance in this sequence}{input[i]}
+ *    If stride_ > 0:
+ *      Output: a shorten sequence. The operation of getting max instance of a
+ *              sequence is independently performed on every slice of the input
```

Comment:
Done. And update the comments in SequencePoolLayer.h, AverageLayer.h and SequenceLastInstanceLayer.h at the same time.